ui/src/scss/_patterns_icons.scss

```diff
@@ -113,6 +113,24 @@
     background-size: 100% 100%;
   }
 
+
+  .p-icon--debug {
+    @extend %icon;
+    background-image: url("data:image/svg+xml,%3Csvg width='16' height='16' viewBox='0 0 16 16' xmlns='http://www.w3.org/2000/svg'%3E%3Ctitle%3Edebug icon 8@2x%3C/title%3E%3Cpath d='M11.673 12.994L14.68 16l1.313-1.316-3.004-3.005A5.501 5.501 0 0 0 8.5 3C5.46 3 3 5.463 3 8.5 3 11.54 5.46 14 8.5 14c1.18 0 2.276-.37 3.173-1.006zM4.25 8.5a4.25 4.25 0 1 1 8.5 0 4.25 4.25 0 0 1-8.5 0zM6 9h5v1H6V9zm0-2h5v1H6V7zM4.71 0C2.845 0 1.646.095.87.87.093 1.648 0 2.847 0 4.716v4.568c0 1.87.094 3.068.87 3.844.667.668 1.68.824 3.13.857v-1.002c-1.327-.044-2.075-.21-2.424-.56-.41-.41-.576-1.318-.576-3.14V4.716c0-1.82.167-2.727.576-3.137C1.986 1.168 2.892 1 4.71 1h4.577c1.82 0 2.726.17 3.135.578.348.35.514 1.097.558 2.422h1.006c-.033-1.45-.19-2.46-.857-3.13C12.352.096 11.153 0 9.286 0H4.71z' fill='gray' fill-rule='evenodd'/%3E%3C/svg%3E");
+    background-size: 100% 100%;
+  }
+
+  .p-icon--audit,
```

Comment:
I don't think we should be mapping the api to css classes. Could we handle this in the component instead, using existing Vanilla classes?

Reply:
Yep, I've mapped these in the component.